meshtastic/powermon/__init__.py

```
@@ -1,11 +1,64 @@
+
+def _missing_optional_backend_class(
+    backend_name: str,
+    dependency_name: str,
+    import_error: ImportError,
+) -> type[PowerSupply]:
+    """Return a placeholder backend class that raises a clear dependency error."""
+
+    class _MissingOptionalBackend(PowerSupply):
+        def __new__(
+            cls, *args: object, **kwargs: object
+        ) -> "_MissingOptionalBackend":
```

Ruff (0.15.4)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)